build/gulpfile.vscode.js

```diff
@@ -440,7 +440,14 @@ function patchWin32DependenciesTask(destinationFolderName) {
 	const cwd = path.join(path.dirname(root), destinationFolderName);
 
 	return async () => {
-		const deps = await glob('**/*.node', { cwd, ignore: 'extensions/node_modules/@parcel/watcher/**' });
+		const deps = await glob('**/*.node',
+			{
+				cwd,
+				ignore: [
+					'extensions/node_modules/@parcel/watcher/**',
+					'**/extensions/microsoft-authentication/**'
```

Comment:
Does the binary artifacts already have proper copyright and versioning, if not this shouldn't be skipped.